### PR TITLE
Feat: casparcg ffmpeg filters

### PR DESCRIPTION
**Pull request by baltedewit**
This PR introduces video and audio filters to the CasparCG implemenation for use with CasparCG 2.3.

_Note that this PR is a based on https://github.com/nrkno/tv-automation-state-timeline-resolver/pull/168, which has its own review process_

### CODE REVIEW

File: src/devices/__tests__/casparcg.spec.ts [hunk 1]
```diff
@@ -187,6 +188,89 @@ describe('CasparCG', () => {
 			seek: 25 * 10
 		})
 	})
+	test('CasparCG: Play AMB for 60s in 50fps', async () => {
+
+		const commandReceiver0: any = jest.fn(() => {
+			return Promise.resolve()
+		})
+		let myLayerMapping0: MappingCasparCG = {
```
Comment (gundelsby) [inline src/devices/__tests__/casparcg.spec.ts:196]: Shouldn't all these be consts?

File: src/devices/casparCG.ts [hunk 2]
```diff
@@ -467,11 +480,11 @@ export class CasparCGDevice extends DeviceWithState<State> implements IDevice {
 				// create a channel in state if necessary, or reuse existing channel
 				const channel = caspar.channels[mapping.channel] || { channelNo: mapping.channel, layers: {} }
 				channel.channelNo = Number(mapping.channel) || 1
-				channel.fps = 25
+				channel.fps = this.initOptions ? this.initOptions.fps || 25 : 25
```
Comment (gundelsby) [inline src/devices/casparCG.ts:483]: Is this project too old to use the optional chaining? If not, this (and similar lines below) reads easier as `channel.fps = this.initOptions?.fps || 25`, no?

Comment (baltedewit) [reply src/devices/casparCG.ts:483]: Yes, the version of Typescript used does not have optional chaining yet

File: src/devices/__tests__/casparcg.spec.ts (hunk 1 above)
Comment (baltedewit) [reply src/devices/__tests__/casparcg.spec.ts]: Yeap, I've changed these in `casparcg.spec.ts` now but when we add better linting we'll have to do this everywhere

**Pull request merged**